### ISSUE TITLE
NVDA does not read the Read-only state in table

### ISSUE THREAD

**Issue by yugimamidi** [nvaccess/nvda#8937]
#### Have you tried any other versions of NVDA? No
[table.txt](https://github.com/nvaccess/nvda/files/2574697/table.txt)

**Comment by Adriani90** [comment]
This is reproducible for every browser.

**Comment by Adriani90** [comment]
It seems NVDA still doesn't support aria readonly="true".
cc: @SaschaCowley I think this might be a quick fix, right?
